app/src/main/java/io/stanwood/mhwdb/feature/armors/ui/ArmorsFragment.kt

```diff
@@ -72,6 +74,9 @@ class ArmorsFragment : Fragment(), HasSupportFragmentInjector {
             .apply {
                 binding = this
                 retryCallback = View.OnClickListener { viewModel.retry() }
+                toWeaponsCallback = View.OnClickListener {
```

Comment:
```suggestion
                toWeaponsCallback = Navigation.createNavigateOnClickListener(NavGraphContainerDirections.showWeapons())
```

Reply:
Ah, neat, didn't know about that one! :)

Though in reality it just supports IDs, but that's still nice.